src/main/java/com/meilisearch/sdk/Documents.java

```diff
@@ -101,15 +104,17 @@ String getDocuments(String uid, int limit, int offset, List<String> attributesTo
      * @param uid Partial index identifier for the document
      * @param document String containing the document to add
      * @param primaryKey PrimaryKey of the document
-     * @return String containing the added document
+     * @return Task MeiliSearch API response
      * @throws Exception if the client request causes an error
      */
-    String addDocuments(String uid, String document, String primaryKey) throws Exception {
-        String requestQuery = "/indexes/" + uid + "/documents";
+    Task addDocuments(String uid, String document, String primaryKey) throws Exception {
+        String urlQuery = "/indexes/" + uid + "/documents";
```

Comment:
Yes, we probably should but like the rest of the code is without I did not add it here.
I will open an issue about it if you think it's a better way to handle it in java

Reply:
No, `StringBuilder` should not be used in this case. It is less readable and gives no advantage in this case, since the compiler will generate efficient code here. You only need `StringBuilder` if the number of strings concatenated is not known at compile time (eg in a loop).

(sorry to jump into the review without being asked, I am just watching this pull request since I use Java and meilisearch 0.25.)

Reply:
Thanks, @uwemaurer!